integration/compute_droplet_action_test.go

```diff
@@ -44,7 +44,7 @@ var _ = suite("compute/droplet-action", func(t *testing.T, when spec.G, it spec.
 				"/v2/droplets/247/actions":     {method: http.MethodPost, body: `{"type":"enable_ipv6"}`},
 				"/v2/droplets/45/actions":      {method: http.MethodPost, body: `{"type":"power_on"}`},
 				"/v2/droplets/1111/actions":    {method: http.MethodPost, body: `{"kernel":7777,"type":"change_kernel"}`},
-				"/v2/droplets/65/actions":      {method: http.MethodPost, body: `{"type":"enable_backups"}`},
+				"/v2/droplets/65/actions":      {method: http.MethodPost, body: `{"backup_policy": {"hour":0, "plan":"daily", "weekday":""},"type":"enable_backups"}`},
```

Comment:
Not sure if this is a valid update

Reply:
Updated here - [d5a31a5](https://github.com/digitalocean/doctl/pull/1609/commits/d5a31a5ec1a2cc688693eb0740e6ac5cc4c9455b) 
Added a check for default backup policy applied when backup policy is not defined on backup enabling.
This prevents from the necessary to change the body of the integration test.